GitHub_Learning/App/SceneDelegate.swift

```diff
@@ -11,45 +11,51 @@ import CleanCore
 import CleanPlatform
 
 class SceneDelegate: UIResponder, UIWindowSceneDelegate {
-
+    
     var window: UIWindow?
-
+    
     func scene(_ scene: UIScene, willConnectTo session: UISceneSession, options connectionOptions: UIScene.ConnectionOptions) {
         guard let windowScene = (scene as? UIWindowScene) else { return }
         self.window = UIWindow(windowScene: windowScene)
-        let initialViewController: OnboardingViewController = try! startAppByResolvingType(scopeSpecProvider: ScopeSpecProvierImpl())
-        self.window?.rootViewController = initialViewController
-        self.window?.makeKeyAndVisible()
+        
+        let onboardingController: OnboardingController = try! startAppByResolvingType(scopeSpecProvider: ScopeSpecProvierImpl())
```

Comment:
Musis tu volat `startAppByResolvingType`? Nejak mi to tu nesedi, ale mozna je to tim, ze ten clean core neznam tak dobre. Nestaci ti tu nejaky instance provider?

Reply:
jsem právě hledal nemohl jsem nic lepšího najít možná, když se na to pak zkusíme společně podívat no..